lib/chewy/search/request.rb

```diff
@@ -22,19 +38,18 @@ class Request
 
       attr_reader :_indexes, :_types, :parameters
 
-      def self.delegated_methods
-        %i[
-          query filter post_filter order reorder docvalue_fields
-          track_scores request_cache explain version profile
-          search_type preference limit offset terminate_after
-          timeout min_score source stored_fields search_after
-          load script_fields suggest indices_boost
-          rescore highlight total total_count total_entries
-          types delete_all count exists? exist? find
-          scroll_batches scroll_hits scroll_results scroll_objects
-        ]
-      end
-
+      # The class is initialized with the list of chewy indexes and/or
+      # types, which are later used for the request composition.
```

Comment:
Could this line be slightly more active? `for the request composition.` => `to compose requests`?